stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/suggestions/mentions/MentionSuggestionItem.kt

```diff
@@ -0,0 +1,155 @@
+package io.getstream.chat.android.compose.ui.messages.suggestions.mentions
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.combinedClickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.compose.R
+import io.getstream.chat.android.compose.ui.common.avatar.UserAvatar
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+
+/**
+ * Represents the default mention suggestion item in the mention suggestion list popup.
+ *
+ * @param user The user that will be used to autocomplete the mention.
+ * @param modifier Modifier for styling.
+ * @param onMentionClick Handler when the user taps on an item.
+ * @param leadingContent Customizable composable function that represents the leading content of a mention item.
+ * @param detailsContent Customizable composable function that represents the center content of a mention item.
+ * @param trailingContent Customizable composable function that represents the trailing content of the a mention item.
+ */
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+public fun DefaultMentionSuggestionItem(
+    user: User,
+    modifier: Modifier = Modifier,
```

Comment:
First non-optioonal parameter - above the leadingContent!

Reply:
Strange, but my Android Studio stopped showing warnings about modifier param position